### PR TITLE
DR-2807: About page

### PR DESCRIPTION
**Pull request by 7emansell**
## Ticket:

Resolves JIRA ticket [DR-2807](https://jira.nypl.org/browse/DR-2807)

## This PR does the following:

- Adds About page
- Installs jwplayer-react
- Fixes link color in nav

## How has this been tested?

Locally

### Checklist:

- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by 7emansell** [comment]
Waiting on confirmation about JWPlayer preview and about a broken DPLA link from Lily but otherwise done

File: src/components/navMenu/navMenu.tsx
Comment (EdwinGuzman) [inline src/components/navMenu/navMenu.tsx:22]: I forgot why this needs to be manually added. This style rule is already applied elsewhere 
Screen Shot 2024-03-20 at 1 04 40 PM

File: src/components/navMenu/navMenu.tsx
Comment (EdwinGuzman) [inline src/components/navMenu/navMenu.tsx:20]: Btw, the Chakra convention is `_hover`. This does the same thing and is the standard CSS convention.

File: __tests__/pages/aboutpage.test.tsx
Comment (EdwinGuzman) [inline __tests__/pages/aboutpage.test.tsx:5]: Move this to the jest setup file so you don't have to redeclare this in other tests.

File: src/pages/about/index.tsx
Comment (EdwinGuzman) [inline src/pages/about/index.tsx:24]: Would using the `TemplateAppContainer` component like the home page help with the static width value? The About page should follow the standard width and now I'm realizing I did not check that detail in the designs before.

File: src/data/aboutData.tsx
Comment (EdwinGuzman) [inline src/data/aboutData.tsx:11]: How about renaming this to include that what is returned are elements? This is not data in the "data" sense like an API data.

File: src/data/aboutData.tsx
Comment (EdwinGuzman) [inline src/data/aboutData.tsx:23]: Why this extra space?

File: src/data/aboutData.tsx
Comment (EdwinGuzman) [inline src/data/aboutData.tsx:70]: Hmm eventually these should be self-referential and be `"/"`, `"/search/index..."`, etc. You could use the `DC_URL` value from appConfig but those urls are also not self-referential. Can you add a TODO comment somewhere so we don't forget? This won't have to be done until this app is entirely complete.

File: src/data/aboutData.tsx
Comment (EdwinGuzman) [inline src/data/aboutData.tsx:493]: My developer brain thought "undefined" was an error.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Looks great but just pointing out minor things. The JWPlayer seems very easy to work with and I saw the issue about the no preview image shown.

Now is also a good time to ask Clare for an accessibility review before this gets merged.

File: src/components/navMenu/navMenu.tsx
Comment (7emansell) [reply src/components/navMenu/navMenu.tsx:22]: Ah it doesn't I thought I was overriding with the `color` and needed to reset

File: src/pages/about/index.tsx
Comment (7emansell) [reply src/pages/about/index.tsx:24]: Yea this was weird. I was assuming (but obviously did not check when reviewing the design 🤦‍♀️) that if the header was on new breakpoints, the width of the content would align. This `820px` width content is in a weird spot between the small tablet breakpoint (`768px`) and the old `lg` breakpoint (`960px`). Bc it's not either, and the page is just this straight up static content, I'm not using the new breakpoints at all as you can see here– all that matters, to create the breakpoint behavior as it is in Figma, is whether the screen width is larger or smaller than the content. Which is all a very long way to say are we gonna keep the content this width knowing that it's not "standard" ?

Comment (EdwinGuzman) [reply src/pages/about/index.tsx:24]: "820px" is directly from Figma, right? We should point it out to Lily and mention we can keep it but it's not standard and this might change when the new breakpoints are in place globally (and not just in the Header).

This _feels_ off to me but this is up to Lily to test and decide.

**Comment by EdwinGuzman** [comment]
The latest updates look good but there's the jwplayer thumbnail issue and still waiting on accessibility review. Will approve after in case you need to make any code updates.

**Comment by 7emansell** [comment]
JWPlayer thumbnail issue fixed– had to put each video into its own playlist on the JWPlayer dashboard for the thumbnails to show up, which was interesting– don't really know where to document that but would like to as I couldn't find it anywhere on the internet. @EdwinGuzman I'll wait for a11y then pass this over to Lily with the width concern noted

**Comment by EdwinGuzman** [comment]
> JWPlayer thumbnail issue fixed– had to put each video into its own playlist on the JWPlayer dashboard for the thumbnails to show up, which was interesting– don't really know where to document that but would like to as I couldn't find it anywhere on the internet. @EdwinGuzman I'll wait for a11y then pass this over to Lily with the width concern noted

This doesn't seem like a dev responsibility to update the JWPlayer dashboard but I understand the need. You could document it in the README but maybe a better place would be confluence for anyone else who has access to the dashboard.

**Review by EdwinGuzman — APPROVED** [review]
I think the updates from vqa may be small enough to not require an approval but I can review again. Just hold off on merging this in until we have a plan for QA/release.

**Review by avertrees — APPROVED** [review]

**Comment by avertrees** [comment]
Approved - when we launch this we'll want to add functionality to link directly to NYPL Statement on Potentially Harmful Content using some sort of slug or query paramter, ie. /about#nypl_harmful_content_statement

**Comment by 7emansell** [comment]
Adding that id, noting that there is the continuing issue of the sticky header actually covering the target element

**Pull request merged**
